extensions/amp-next-page/amp-next-page.md

```diff
@@ -64,36 +68,151 @@ The pages should be inlined using a JSON format.
+classes:
+
+- `.amp-next-page-links` for the containing element
+- `.amp-next-page-link` for an individual link
+- `.amp-next-page-image` for the link image
+- `.amp-next-page-text` for the link text
+
+### Separator
+
+A separator is rendered between each loaded document. By default this is 
+rendered as a full-width hairline. It can be customised by styling the `
+.amp-next-page-default-separator` class.
+
+Alternatively, you can specify a custom separator containing arbitrary HTML 
+content as a child of the `amp-next-page` component by using the `separator` 
+attribute.
+```html
+<amp-next-page src="https://example.com/config.json">
+  <div separator>
+    <h1>Keep reading</h1>
+  </div>
+</amp-next-page>
+```
+
+### Element hiding
+
+Elements which are common across multiple pages can be programmatically 
+hidden in child documents, to avoid e.g. stacking up multiple page footers at
+the end of the document, or to hide the page header from each subsequent page.
+
+Elements can be hidden by specifying one or more string CSS selectors in the 
+`hideSelectors` key of the element config. Elements matching any of the 
+selectors will be set to `display: none` in all child documents.
+```html
+<amp-next-page>
+  <script type="application/json">
+    {
+      "hideSelectors": [
+        ".header",
+        ".main footer",
+        "#navigation"
+      ],
+      "pages": ...
+    }
+  </script>
+</amp-next-page>
+```
 
 ## Attributes
 
-N/A
+##### src (required)
+
+The URL of the remote endpoint that returns the JSON that will be used to 
+configure this `amp-next-page` component. This must be a CORS HTTP service. 
+The URL's protocol must be HTTPS.
+
+{% call callout('Important', type='caution') %} Your endpoint must implement 
+the requirements specified in the CORS Requests in AMP spec. {% endcall %}
+
+The `src` attribute may be omitted if the config has been specified inline.
 
 ##### Common attributes
 
-This element includes [common attributes](https://www.ampproject.org/docs/reference/common_attributes) extended to AMP components.
+This element includes 
+[common attributes](https://www.ampproject.org/docs/reference/common_attributes) 
+extended to AMP components.
 
 ## Configuration Spec
 
-The configuration defines the documents recommended by `<amp-next-page>` to the user.
+The configuration defines the documents recommended by `amp-next-page` to 
+the user as a JSON object.
+
+| Key                | Value |
+| ------------------ | ----- |
+| `pages` (required) | Ordered array of one or more page objects |
+| `hideSelectors`    | Optional array of string CSS selectors to hide in child documents |
+
+Each page object should have the following format:
+
+| Key                   | Value |
+| --------------------- | ----- |
+| `ampUrl` (required)   | String URL of the page. Must be on the same origin as the current document. URLs will automatically be rewritten to point to the CDN when being served from the cache. |
```

Comment:
Should use "Google AMP Cache" instead of "CDN" and "cache"   (might have to find a way to phrase differently to avoid repetition)

Reply:
Done.